src/palettes/dracula.json

```diff
@@ -0,0 +1,15 @@
+{
+ "Dracula": {
+    "Background": "#282a36",
+    "Selection": "#44475a",
+    "Foreground": "#f8f8f2",
+    "Comment": "#6272a4",
+    "Cyan": "#8be9fd",
+    "Green": "#50fa7b",
+    "Orange": "#ffb86c",
+    "Pink": "#ff79c6",
+    "Purple": "#bd93f9",
+    "Red": "#ff5555",
+    "Yellow": "#f1fa8c"
+ }
+}
```

Comment:
Please try formatting this file to look similar to the other palettes:
- Indent size = 4
- Uppercase hex color